docs/copied-from-beats/running-on-docker.asciidoc

```diff
@@ -0,0 +1,61 @@
+[[running-on-docker]]
+=== Running {beatname_uc} on Docker
+
+Docker images for {beatname_uc} are available from the Elastic Docker
+registry. You can retrieve an image with a `docker pull` command.
+
+ifeval::["{release-state}"=="unreleased"]
```

Comment:
Is `release-state` defined somewhere?

Reply:
They are defined and included from here: https://github.com/elastic/apm-server/blob/master/_beats/libbeat/docs/version.asciidoc

Before we backport this we'll need to override those to match our 6.0